THIRD_PARTY_LICENSES.txt

```diff
@@ -4786,7 +4786,7 @@ github.com/docker/go-events
 Copyright © 2016 Docker, Inc. go-events is licensed under the Apache License,
 
 -------- Dependency
-github.com/docker/go-metrics [uses Creative Commons license for doc - see CC license at end of file]
```

Comment:
All this Creative Commons license stuff needs to be added back.  My understanding is that the attribution-helper does not include it and manual intervention is needed.

Reply:
resolved in last commit